Added diagrams in `Spot Price Graph` Lesson

AAYUSH-GUPTA-coder opened the pull request:
Added desmos diagrams for `Initial Point: (X,Y)` and `New Point: (X + dX, Y + dY)`

Equious reviewed (changes requested):
Hey! Thanks for this. We'd like to host the images, could you adjust this submission to include the images in the `static` folder? Might need a new directory for this lesson.

Then in the markdown we can use image tags to reference it

<img src="static/uniswap-v2/etc">

AAYUSH-GUPTA-coder commented:
Is this the link to the [advanced-defi-2024](https://github.com/Cyfrin/advanced-defi-2024) course imges?

https://github.com/Cyfrin/Updraft/tree/main/static/foundry-defi

If yes, could you please suggest the name of the folder for the diagrams?

If no, could you please share the link to the correct folder?

AAYUSH-GUPTA-coder commented:
@Equious  I've made the requested changes:

- Created a new folder named "uniswap-v2" in the "static" folder.
- Moved all relevant images into this folder.
- Updated the image references in the lesson to use the new path.

Could you please review these changes and let me know if any further adjustments are needed? Thank you for your guidance.

Equious reviewed (approved):
Looks good! Thanks for the contribution! 🙏
